src/main/index.ts

```diff
@@ -658,30 +652,23 @@ const router = t.router({
 
     return !isInitializedValidated;
   }),
-  holochainVersion: t.procedure.query(
-    () => HOLOCHAIN_MANAGERS[Object.keys(HOLOCHAIN_MANAGERS)[0]].version,
-  ),
-  isDevhubInstalled: t.procedure.query(() =>
-    Object.values(HOLOCHAIN_MANAGERS)
-      .flatMap((manager) => manager.installedApps)
-      .some((app) => app.installed_app_id === DEVHUB_APP_ID),
-  ),
+  holochainVersion: t.procedure.query(() => HOLOCHAIN_MANAGERS[DEFAULT_HOLOCHAIN_VERSION].version),
+  isDevhubInstalled: t.procedure.query(() => isDevhubInstalled(HOLOCHAIN_MANAGERS)),
   getInstalledApps: t.procedure.query(() => {
     const filterHeadlessApps = (app: { installed_app_id: string }) =>
       ![DEVHUB_APP_ID, APP_STORE_APP_ID].includes(app.installed_app_id);
-    const mapAppInfo = (manager: HolochainManager) => (app: AppInfo) => {
-      const icon = manager.appIcon(app.installed_app_id);
+    const defaultManager = HOLOCHAIN_MANAGERS[DEFAULT_HOLOCHAIN_VERSION];
```

Comment:
It looks like `getInstalledApps` can now only return apps installed in the default holochain version but that kind of defeats the purpose of having multiple HolochainManagers in the first place. So I think we need to change the logic here that it still always returns apps of all holochain versions and only in case of the default holochain version it filters out devhub and appstore.